instrumentation/spring/spring-boot-autoconfigure/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+description: >
+  This instrumentation auto-configures OpenTelemetry instrumentation for spring-web, spring-webmvc,
```

Comment:
most users use it together with https://opentelemetry.io/docs/zero-code/java/spring-boot/